index.css

```diff
@@ -0,0 +1,171 @@
+html body {
+    font-family: Helvetica, sans-serif;
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+    background-color: black;
+    color: rgba(255, 246, 240, 0.9);
+}
+
+nav {
+    display: inline-flex;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
Notice many times you need to call this color? What if you wanted to change the color? You would have to go through the **entire** website and change **every** instance of it. This is why in programming you follow a rule called **DRY**, or **D**on't **R**epeat **Y**ourself. 

In a small project like this, I would reccomend [defining CSS variables under the root and calling them throughout the stylesheet](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

```css
:root {
  --sea-shell-color: rgba(255, 246, 240, 0.9);
}

body {
    color: var(--sea-shell-color);
}

nav {
    border-bottom: solid var(--sea-shell-color) 1px;
}
```